readme.md

```diff
@@ -23,13 +23,10 @@ e.g.: you can create a google calendar for your company to share Meetup events w
 
 ## Installation ##
 
-### Get meetsy ###
-You can clone or download meetsy by using the "Clone or Download" Button above.
-
 ### Install meetsy ###
 
 ```node
-npm install
+npm install -g meetsy
```

Comment:
Now you also have to change the sentence "save this data at project root and rename it to googleApi_clientSecret.json" in the bullet points, maybe something like: "save this JSON file anywhere for later use as `--secret` (remember the path to the file!)"

Reply:
dammit...
thank you so much

Reply:
ready